controllers/PostController.php

```diff
@@ -111,7 +111,7 @@ public function fetchUserPosts($userId, $pid = '')
 
     if($pid == '')
     {
-      $sql  = "SELECT * from posts WHERE uid = $userId ORDER by pid DESC";
+      $sql  = "SELECT po.*,us.followers as followers from posts po INNER JOIN users us ON po.uid = us.id WHERE us.id = $userId"; 
```

Comment:
Make it more easy. 

`"SELECT po.*,us.followers as followers, sum(po.likes) as totalLikes, sum(po.shares) as totalShares  from posts po INNER JOIN users us ON po.uid = us.id WHERE us.id = $userId";`